src/UI/Component/Card/Card.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/* Copyright (c) 2016 Timon Amstutz <timon.amstutz@ilub.unibe.ch> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Card;
+
+/**
+ * Interface Card
+ * @package ILIAS\UI\Component\Card
+ */
+interface Card extends \ILIAS\UI\Component\Component {
```

Comment:
Does this model fit the description in the Factory? Looks as if i could have a title, an image and a header section. This seems to be odd, or i do not understand the description correctly. Can the header section contain another title and image? How would that be rendered without looking weird?